src/components/Keyword/AddKeyword/styles.js

```diff
@@ -0,0 +1,292 @@
+  top: 335px;
+  left: ${(props) => (props.toggle ? '588px' : '453px')};
+  width: 1092px;
+  height: 117px;
+  background-color: #eee;
+  padding: 10px 12px;
+`;
+
+export const SiteList = styled.ul`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+export const SiteItem = styled.li`
+  height: 21px;
+  display: flex;
+  align-items: center;
+`;
+
+export const SiteName = styled.span`
+  margin-right: 4px;
+`;
+
+export const CloseBtn = styled.button`
+  display: flex;
+  align-items: center;
+  margin-right: 36px;
+`;
+
+export const XImage = styled.img``;
+
+export const ImportantContainer = styled.div`
+  height: 16px;
+  display: flex;
+  position: absolute;
+  left: 590px;
+  top: 501px;
+  align-items: center;
+  cursor: pointer;
+`;
+
+export const NormalContainer = styled(ImportantContainer)`
+  top: 530px;
+  cursor: pointer;
+`;
+
+export const CheckBox = styled.div`
+  min-width: 14px;
+  height: 14px;
+  margin-right: 8px;
+  border-radius: 3px;
+  border: 1px solid ${theme.colors.sliver};
+  cursor: pointer;
+
+  ${(props) => {
+    if (props.isNormalAlarm || props.isImportantAlarm || props.isVibrationAlarm || props.isSlientAlarm) {
+      return `
+                background-image:url('/asset/check.svg');
+                background-position:center;
+            `;
+    } else {
+      return `
+                background-image:none;
+            `;
+    }
+  }}
+`;
+
+export const CheckBoxTitle = styled.span`
+  min-width: 55px;
+  font-size: 14px;
+  color: #999999;
+  margin-right: 24px;
+  cursor: pointer;
+`;
+
+export const CheckBoxContent = styled.span`
+  min-width: 241px;
+  font-size: 11px;
+  color: #999999;
+`;
+
+export const SettingContainer = styled.div`
+  width: 1092px;
+  height: 159px;
+  position: absolute;
+  border: 1.5px solid #eee;
+  padding-left: 24px;
+  left: 588px;
+  top: 570px;
+  display: flex;
+  flex-direction: column;
+`;
+
+export const ModeContainer = styled.div`
+  width: 100%;
+  display: flex;
+  align-items: center;
+  margin: 16px 0px 32px 0px;
+`;
+
+export const SlientMode = styled(CheckBoxTitle)`
+  margin-right: 8px;
+`;
+
+export const SlientCheckBox = styled(CheckBox)`
+  margin-right: 32px;
+`;
+
+export const VibrationCheckBox = styled(CheckBox)`
+  margin-right: 24px;
+`;
+
+export const SettingContent = styled(CheckBoxContent)``;
+
+export const AlarmContainer = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+`;
+
+export const AlarmTitle = styled.span`
+  margin-bottom: 16px;
+`;
+
+export const AlarmType = styled.ul`
+  display: flex;
+`;
+
+export const Type = styled.li`
+  cursor: pointer;
+  margin-right: 48px;
+  color: ${(props) => {
+    if (props.checkId === props.alarmTerm) {
+      return `${theme.colors.darkgray}`;
+    } else {
+      return `${theme.colors.silver}`;
+    }
+  }};
+`;
+
+export const EditButton = styled.button`
+  width: 80px;
+  height: 32px;
+  background: ${theme.colors.darkgray};
```

Comment:
상단에 theme import 받아서 써도 되지만 ThemeProvier 로 이미 제공을 하고 있어서 props로 부터 가져올 수 있습니다.

```suggestion
  background: ${(props) => props.theme.colors.darkgray};
```